workflows/phaseimpute/main.nf

```diff
@@ -155,9 +155,9 @@ workflow PHASEIMPUTE {
             VCF_PHASE_SHAPEIT5(
                 VCF_NORMALIZE_BCFTOOLS.out.vcf_tbi.combine(Channel.of([[]])),
```

Comment:
Can we simplify `Channel.of([[]])` as well?

Reply:
Apparently we cannot 😉